<?xml version="1.0"?>
<launch>
    <node pkg="runbot_routing_machine" type="routing_machine" name="routing_machine" output="screen" cwd="node">
        <param name="algorithm" value="MLD" type="string" />
        <param name="server_type" value="local" type="string" />
    </node>
    <node pkg="runbot_routing_machine" name="routing_machine_route_handler" type="routing_machine_route_handler.py" respawn="true" output="screen"/>
</launch>